dispatch/theme/fields.py

```diff
@@ -176,16 +200,55 @@ class EventField(ModelField):
     model = Event
     serializer = EventSerializer
 
-
 class WidgetField(Field):
 
     type = 'widget'
 
+    def __init__(self, label, zone, required=False):
+        super(WidgetField, self).__init__(label, required=required)
+        self.zone_id = zone.id
```

Comment:
How would you feel about passing in a list of widgets instead of associating widget fields with a "nested" zone? If all we are really trying to do is specify a list of compatible widgets for the field, I don't think we have to use a zone.

Would this add a lot of complexity to the current design?